### ISSUE TITLE
[UX] Move non-displayed pseudo fields to vertical tab or group on "Manage fields" page

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#4002]
**Description of the need**
Pseudo fields clutter up the "Manage Fields" page. See https://github.com/backdrop-contrib/xmlsitemap/issues/56

![screenshot-2018-6-28 feature packweb](https://user-images.githubusercontent.com/2385329/42030184-a739fd82-7ada-11e8-81d3-d8c2e5fb0601.png)

**Proposed solution**
Can we move these fields all to a vertical tab, or at least group them separately somehow, to differentiate them from the regular fields while still allowing re-sorting?

**Comment by BWPanda** [comment]
The problem is that you'll often want to sort them with regular fields (e.g. display a pseudo field between two regular fields). You wouldn't be able to do that if you grouped/moved them separately.

**Comment by laryn** [comment]
@BWPanda 
Maybe I'm not thinking of when you'd want to sort one of these between a regular field, or how you'd be able to. Don't they always go to the bottom of the page already, no matter where they are sorted here? (e.g. XML Sitemap doesn't show as a field but shows in the tabs at the bottom no matter where it is positioned on "Manage Fields", doesn't it?)

**Comment by BWPanda** [comment]
Yes, XML Sitemap doesn't render as a normal, weighted field because it's displayed in vertical tabs. A better example is the Submitted By pseudo field. Not only is it rendered as a normal field, but you generally don't want it displayed at the bottom and need the freedom to move it around among other fields.

**Comment by laryn** [comment]
Is there a way to filter the ones that don't render out of the list and group them at the bottom on the Manage Fields page?
